src/hb-subset-plan.cc

```diff
@@ -314,22 +314,16 @@ static void _colr_closure (hb_face_t *face,
   OT::COLR::accelerator_t colr (face);
   if (!colr.is_valid ()) return;
 
-  unsigned iteration_count = 0;
   hb_set_t palette_indices, layer_indices;
-  unsigned glyphs_num;
-  {
-    glyphs_num = glyphs_colred->get_population ();
-    // Collect all glyphs referenced by COLRv0
-    hb_set_t glyphset_colrv0;
-    for (hb_codepoint_t gid : glyphs_colred->iter ())
-      colr.closure_glyphs (gid, &glyphset_colrv0);
-
-    glyphs_colred->union_ (glyphset_colrv0);
-
-    //closure for COLRv1
-    colr.closure_forV1 (glyphs_colred, &layer_indices, &palette_indices);
-  } while (iteration_count++ <= HB_CLOSURE_MAX_STAGES &&
-           glyphs_num != glyphs_colred->get_population ());
+  // Collect all glyphs referenced by COLRv0
+  hb_set_t glyphset_colrv0;
+  for (hb_codepoint_t gid : glyphs_colred->iter ())
```

Comment:
I think you don't need the `->iter ()` here.

Reply:
there's an error without `->iter()`: error: no matching function for call to ‘begin(hb_set_t*&)’
Seems either `+hb_iter ()` or `->iter () `would work.

Reply:
Ah it's a pointer. Then just `*glyphs_colred`.

Reply:
Done. Thank you!